### ISSUE TITLE
does not support sandboxes with company names

### ISSUE THREAD

**Issue by hlopetz** [sorenkrabbe/Chrome-Salesforce-inspector#173]
works well with company.lightning.force.com URLs, but with company--sandbox.lightning.force.com ones neither detects sObject type, nor shows "Show all data" button.

![изображение](https://user-images.githubusercontent.com/645340/92638843-9f293f80-f2e3-11ea-8699-dae88506e9a3.png)

**Comment by iscariot0ne** [comment]
Having the same issue. I hope this can be fixed soon, it makes my life so much easier!

**Comment by hlopetz** [comment]
yeah, seems something between the extension and SF corrupts website/plugin data. since i see similar behavior in Firefox, browsers aren't guilty.

**Comment by tomasrgar** [comment]
Thank you very much for the workoarund, @iscariot0ne. It works for me. I think this could be caused by a critical update in this summer that remove the instance name for sandboxes with custom domain.

Stabilize the Hostname for My Domain URLs in Sandboxes - Critical Update

Maybe some custom cookies have been stored with the old URL and that could be the problem.
